roles/docker/tasks/package_docker.yml

```diff
@@ -117,6 +117,18 @@
   notify:
   - restart docker
 
+- name: Check for credentials file for registry auth
+  stat:
+    path: "{{ docker_cli_auth_config_path }}/config.json"
+  when: oreg_auth_user is defined
+  register: docker_cli_auth_credentials_stat
+
+- name: Create credentials for docker cli registry auth
```

Comment:
We need to create these credentials in /root/.docker/config.json to enable openshift_version to pull down the base container image.  Additionally, having this populated on each container host is necessary for those hosts to pull down containers to themselves.